requirements.txt

```diff
@@ -1,6 +1,6 @@
 pydantic==2.9.2
 websockets==13.1
 protobuf==5.29.3
-web3==7.7.0
-eth-utils
-eth-account
+web3[tester]==7.7.0
```

Comment:
If something is needed only for development/testing, please add requirement to requirements**-dev.txt** file

Reply:
I've added `eth_tester` to requirements-dev.txt, feel free to remove it if `web3[tester]` duplicates requirement.